lib/topology_lib_iperf/library.py

```diff
@@ -102,11 +120,18 @@ def client_start(
     action is taken for that parameter (left "as-is").
     :param enode: Engine node to communicate with.
     :type enode: topology.platforms.base.BaseNode
-    :param server_ip: Server's IP address in the form ``'192.168.1.10'``
-    :param int port: iperf port to be open
-    :param int interval: interval for iperf server to check
-    :param int time: the time iperf client will be running
+    :param server_ip: Server's IP address in the form ``'192.168.1.10'``.
+    :param int port: iperf port to be open.
+    :param int interval: interval for iperf server to check.
+    :param int time: the time iperf client will be running.
     :param bool udp: If it is UDP or TCP. Default is False for TCP.
+    :param str bandwidth: Bandwidth for iperf to use in bits/sec.
+     When used automatically switches to UDP regardless of udp setting.
+     Default is 1Mb/sec.
```

Comment:
What I'm going to do here is remove the default note, which is the default for iperf. If I set the default as part of the argument iperf will start malfunctioning since the bandwidth option switches automatically to UDP regardless of what the user has set.